lib/msal-browser/src/crypto/BrowserCrypto.ts

```diff
@@ -177,6 +207,44 @@ export class BrowserCrypto {
         });
     }
 
+    /**
+     * IE Helper function for importKey
+     * @param key 
+     * @param format 
+     * @param extractable 
+     * @param usages 
+     */
+    private async msCryptoImportKey(keyBuffer: ArrayBuffer, extractable: boolean, usages: Array<KeyUsage>): Promise<CryptoKey> {
```

Comment:
Should all of the IE stuff be its own file?

Reply:
I can separate it into it's own file for readability, I thought it would be easier to have all the crypto APIs in a single file.